pages/plugins/decorators.rst

```diff
@@ -0,0 +1,63 @@
+.. _decorators_api:
+
+**********
+Decorators
+**********
+
+:ref:`decorators` can be used to transform a message field value during searches. Multiple decorators can be applied at the same time, but you cannot make any assumptions about their order, as that is user defined.
```

Comment:
"transform a message field value during searches" &rarr; "transform a message field at display time"

"During searches" might be misinterpreted to "at search time".